admin/app/views/workarea/admin/shipping_skus/index.html.haml

```diff
@@ -0,0 +1,81 @@
+- @page_title = t('workarea.admin.shipping_skus.index.title')
+
+.view
+  .view__header
+    .grid.grid--middle.grid--center
```

Comment:
note: didn't need that spacer boi here because there was nothing to align left/right, just center.